Library/Formula/qjackctl.rb

```diff
@@ -0,0 +1,62 @@
+class Qjackctl < Formula
+  desc "simple Qt application to control the JACK sound server daemon"
+  homepage "http://qjackctl.sourceforge.net"
+  url "https://downloads.sourceforge.net/qjackctl/qjackctl-0.4.1.tar.gz"
+  sha256 "98024e1ea9f55ac16c290feda051d6494b2261e83f6a918f0fa583b509e2bfd1"
+
+  depends_on "qt"
+  depends_on "jack"
+  depends_on :x11
```

Comment:
If this is a mandatory dependency, I'm afraid it'll have to go in our homebrew/x11 tap.